tests/sentry/event_manager/test_event_manager.py

```diff
@@ -120,6 +120,52 @@ def make_release_event(self, release_name: str, project_id: int) -> Event:
 
 
 class EventManagerTest(TestCase, SnubaTestCase, EventManagerTestMixin, PerformanceIssueTestCase):
+    def test_group_in_deletion_in_progress_should_not_create_new_group(self) -> None:
```

Comment:
~Wait, isn't this the opposite of the behavior you're trying to promote?~

Oh, this is with the feature flag off.

```suggestion
	@with_feature({"organizations:group-deletion-in-progress", False})
    def test_group_in_deletion_in_progress_should_not_create_new_group_when_flag_off(self) -> None:
```